broad/broad/settings.py

```diff
@@ -0,0 +1,19 @@
+BOT_NAME = 'broad'
+
+SPIDER_MODULES = ['broad.spiders']
+NEWSPIDER_MODULE = 'broad.spiders'
+
+CLOSESPIDER_ITEMCOUNT = 100000
+RETRY_ENABLED = False
+COOKIES_ENABLED = True
+
+LOGSTATS_INTERVAL = 3
+LOG_LEVEL = 'INFO'
+MEMDEBUG_ENABLED = True
+CONCURRENT_REQUESTS = 120
```

Comment:
I think it makes sense to enable [autothrottle extension](https://doc.scrapy.org/en/latest/topics/autothrottle.html#std:setting-AUTOTHROTTLE_ENABLED) as it allows to crawl multiple sites politely and not tune any delays for them. Default settings are fine I think, so adding ``AUTOTHROTTLE_ENABLED = True`` would be enough.

Also, scrapy docs recommend increasing ``REACTOR_THREADPOOL_MAXSIZE`` to 20 [here](https://doc.scrapy.org/en/latest/topics/broad-crawls.html#increase-twisted-io-thread-pool-maximum-size) when doing broad crawling.